core/unix/loader.c

```diff
@@ -1147,12 +1147,7 @@ privload_call_lib_func(dcontext_t *dcontext, privmod_t *privmod, fp_t func)
      */
     dummy_argv[0] = dummy_str;
     dummy_argv[1] = NULL;
-    TRY_EXCEPT_ALLOW_NO_DCONTEXT(
```

Comment:
This was here for a reason: I don't think we can safely just remove it.  There is also no explanation given as to *why* this should be removed.  Especially with the recent glibc updates where private lib function crashes are more likely yet are sometimes inconsequential, we'd prefer to not die and instead give a warning.